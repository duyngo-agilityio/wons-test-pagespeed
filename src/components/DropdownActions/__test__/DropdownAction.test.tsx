import Dropdown from '../index';

describe('Dropdown Component', () => {
  it('should match the snapshot', () => {
    const { container } = testLibJestUtils.render(
      <Dropdown id={1} onEdit={jest.fn()} onDelete={jest.fn()} />,
    );

    expect(container).toMatchSnapshot();
  });
});
