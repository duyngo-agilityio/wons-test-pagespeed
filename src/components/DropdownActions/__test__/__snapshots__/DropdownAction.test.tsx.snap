// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dropdown Component should match the snapshot 1`] = `
<div>
  <div
    data-overlay-container="true"
  >
    <script>
      !function(){try{var d=document.documentElement,c=d.classList;c.remove('light','dark');var e=localStorage.getItem('theme');if('system'===e||(!e&&false)){var t='(prefers-color-scheme: dark)',m=window.matchMedia(t);if(m.media!==t||m.matches){d.style.colorScheme = 'dark';c.add('dark')}else{d.style.colorScheme = 'light';c.add('light')}}else if(e){c.add(e|| '')}else{c.add('light')}if(e==='light'||e==='dark'||!e)d.style.colorScheme=e||'light'}catch(e){}}()
    </script>
     
    <button
      aria-expanded="false"
      aria-haspopup="true"
      class="group relative inline-flex items-center justify-center box-border appearance-none select-none whitespace-nowrap font-normal overflow-hidden tap-highlight-transparent data-[pressed=true]:scale-[0.97] outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 px-4 min-w-20 h-10 text-small gap-2 rounded-medium [&>svg]:max-w-[theme(spacing.8)] transition-transform-colors-opacity motion-reduce:transition-none text-default-foreground data-[hover=true]:opacity-hover z-10 aria-expanded:scale-[0.97] aria-expanded:opacity-70 subpixel-antialiased border-none bg-bone"
      data-slot="trigger"
      id="react-aria-:r3:"
      type="button"
    >
      <svg
        class="text-blue-500"
        fill="currentColor"
        height="24"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 512 512"
        width="24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M328 256c0 39.8-32.2 72-72 72s-72-32.2-72-72 32.2-72 72-72 72 32.2 72 72zm104-72c-39.8 0-72 32.2-72 72s32.2 72 72 72 72-32.2 72-72-32.2-72-72-72zm-352 0c-39.8 0-72 32.2-72 72s32.2 72 72 72 72-32.2 72-72-32.2-72-72-72z"
        />
      </svg>
    </button>
    <div
      class="fixed top-4 right-4"
    />
  </div>
</div>
`;
